### PR TITLE
Integration bench test support on exist database file.

### PR DESCRIPTION
**Pull request by laodouya**
Update GNTE test script, add all/4/2/1 goroutine test
Update GNTE submodule to newest version

### CODE REVIEW

File: cmd/cql-minerd/integration_test.go [hunk 1]
```diff
@@ -409,76 +416,69 @@ func benchDB(b *testing.B, db *sql.DB, createDB bool) {
...
 		b.ResetTimer()
 		b.RunParallel(func(pb *testing.PB) {
 			for pb.Next() {
-				i := atomic.AddInt32(&i, 1)
-				index := int(i)%insertedCount + int(start) + 1
-				row := db.QueryRow("SELECT nonIndexedColumn FROM test WHERE indexedColumn = ? LIMIT 1", index)
-				var result int
+				var index int64
+				if createDB { //only data by insert
+					index = rand.Int63n(count-1) + ROWSTART
```
Comment (auxten) [inline cmd/cql-minerd/integration_test.go:460]: gofmt this

File: cmd/cql-minerd/integration_test.go [hunk 2]
```diff
@@ -409,76 +416,69 @@ func benchDB(b *testing.B, db *sql.DB, createDB bool) {
...
+	b.Run("benchmark INSERT", func(b *testing.B) {
 		b.ResetTimer()
-		insertedCount = b.N
 		b.RunParallel(func(pb *testing.PB) {
 			for pb.Next() {
-				ii := atomic.AddInt32(&i, 1)
-				_, err = db.Exec("INSERT INTO test ( indexedColumn, nonIndexedColumn ) VALUES"+
-					"(?, ?)", start+ii, ii,
+				ii := atomic.AddInt64(&i, 1)
+				_, err = db.Exec("INSERT INTO insert_table0 ( k, v1 ) VALUES"+
```
Comment (auxten) [inline cmd/cql-minerd/integration_test.go]: you hava `const TABLENAME = "insert_table0"` defined, so use it

File: cmd/cql-minerd/integration_test.go (hunk 1 above)
Comment (laodouya) [reply cmd/cql-minerd/integration_test.go:460]: gofmt did this... Maybe some complex format rules.

**Review by auxten — APPROVED** [review]
LGTM

**Pull request merged**